data/categories.txt

```diff
@@ -2259,7 +2259,7 @@ nl:^Kerk|tempel|kathedraal|gebedsplaats|gebedshuis
 fi:^Kirkko|uskonnollinen kohde
 fr:^Église|lieu de culte|temple|basilique
 de:^Kirche|Dom|3Kathedrale|4Münster|Anbetungsstätte|Tempel
-hu:^Templom
+hu:^Templom|Székesegyház|Katedrális|Dóm|Bazilika|Istentisztelei hely
```

Comment:
Should the last one be "Istentiszteleti hely"?

Reply:
Yes, you're right. Thanks for spotting!